src/VolleyManagement.Backend/VolleyManagement.Domain/GamesAggregate/GameResultDto.cs

```diff
@@ -63,9 +63,9 @@ public class GameResultDto
         public byte GameNumber { get; set; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether it is allowed to edit game's result (for Playoff scheme)
+        /// Gets or sets two values indicating if game is in PlayOff tournament and is allowed to edit all except total score.
         /// </summary>
-        public bool AllowEditResult { get; set; }
+        public (bool isPlayOff, bool allowEditExceptTotalScore) AllowEditResult { get; set; }
```

Comment:
I feel like isPlayoff and allowEditExceptTotalScore are mutually exclusive

Reply:
it is good, we know if game is in playoff and if next game has results, also I think that we will need that isPlayoff in future

Reply:
why to say allowEditExceptTotalScore? if it is a combination of allowEdit and isPlayoff?

Reply:
because in Playoff if next game has results will be (true,true) and if next game does not have results it will be (true,false),
and for nonplayoff it by default is (false, false)

Reply:
So for non play off it will disable editing?

Reply:
no, now it full corresponds to issue reqierements